src/main/java/ch/njol/skript/util/slot/DroppedItemSlot.java

```diff
@@ -48,7 +48,17 @@ public void setItem(@Nullable ItemStack item) {
 		assert item != null;
 		entity.setItemStack(item);
 	}
-
+	
+	@Override
+	public int getAmount() {
+		return entity.getItemStack().getAmount();
+	}
+	
+	@Override
+	public void setAmount(int amount) {
+		entity.getItemStack().setAmount(amount);
```

Comment:
Does this actually set the amount of the ItemStack? Bukkit has this habit of passing a clone of the original object instead of the reference to keep immutability, kinda stupid but it's done practically everywhere so just making sure.

Reply:
When I tested, it worked.

Reply:
If you so feel, I could get item stack, set amount, and set back the item stack, just for safety and the random event something changes in the future.

Reply:
So Bukkit is actually passing the reference, it should be fine then, if they change this behaviour it's up to them at that point aynways so we can leave it this way.